src/test/java/com/conveyal/gtfs/validator/TravelTooFastTest.java

```diff
@@ -0,0 +1,57 @@
+package com.conveyal.gtfs.validator;
+
+import com.conveyal.gtfs.TestUtils;
+import com.conveyal.gtfs.loader.FeedLoadResult;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.sql.DataSource;
+
+import static com.conveyal.gtfs.GTFS.*;
+import static com.conveyal.gtfs.TestUtils.assertThatSqlCountQueryYieldsExpectedCount;
+
+public class TravelTooFastTest {
```

Comment:
I think the convention we generally use for these test classes is to just name them for the validator, e.g., `SpeedTripValidatorTest.java`.

Reply:
Updated.